.github/workflows/check-spec.yml

```diff
@@ -22,11 +22,11 @@ jobs:
 
     - name: Get the changed files
       run: |
-        echo "Files changed: '$(git diff-tree --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }})'"
-        changed_specs=$(git diff-tree --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }} | { grep "\.spec$" || test $? = 1; })
+        echo "Files changed: '$(git diff-tree --no-commit-id --name-status -r ${{ env.base_sha }} ${{ github.sha }})'"
+        changed_specs=$(git diff-tree --no-commit-id --name-only --diff-filter=M -r ${{ env.base_sha }} ${{ github.sha }} | { grep "\.spec$" || test $? = 1; })
```

Comment:
If I understood [the documentation](https://git-scm.com/docs/git-diff-tree) correctly, this will only give us modified specs and I'm not sure we'll get the ones, which were moved + modified, not to mention new ones. I'd suggest using `--diff-filter=d` to exclude deleted files instead.

Reply:
Taking all spec files except deleted ones.